code/modules/vehicles/spacepod/movement.dm

```diff
@@ -0,0 +1,106 @@
+// brakes, or autostabilize if not driven
+/obj/vehicle/sealed/space_pod/process()
+	process_huds()
+	if(isnull(drift_handler))
+		return
+
+	var/list/drivers = return_drivers()
+
+	if(drivers)
+		var/braking = FALSE
+		for(var/mob/driver as anything in drivers)
+			if(driver.client?.keys_held["Shift"])
+				braking = TRUE
+				break
+
+		if (!braking)
+			return
+
+	//braking without drivers is half as strong incase you put like a bomb in your trunk or something and jumped out
+	drift_handler.stabilize_drift(target_force = 0, stabilization_force = !length(drivers) ? stabilizer_force / 2 : stabilizer_force)
+
+/obj/vehicle/sealed/space_pod/has_gravity(turf/gravity_turf)
+	return FALSE //this proc only exists so movement is better
+
+/obj/vehicle/sealed/space_pod/Move(turf/newloc, direct, glide_size_override)
+	if(!isturf(newloc) || newloc.density || ispodpassable(newloc) || (!newloc.has_gravity() && ispodpassable_nograv(newloc)))
+		return ..()
+	if(newloc.is_blocked_turf()) //weird silly hack to allow us to ram objects on inaccessible turfs
+		var/atom/target
+		for(var/atom/movable/thing as anything in newloc.contents)
+			if(isnull(target) || ((thing.layer > target.layer || thing.flags_1 & ON_BORDER_1) && !(target.flags_1 & ON_BORDER_1)))
+				target = thing
+		Bump(target)
+
+/obj/vehicle/sealed/space_pod/vehicle_move(direction)
+	. = ..()
```

Comment:
why are you calling parent all it does is return false, should just do that explicitly here